app/packs/stylesheets/decidim/decidim_application.scss

```diff
@@ -5,3 +5,89 @@
 // To override CSS variables or Foundation settings use _decidim-settings.scss
 //
 // By default this is empty.
+
+.main-header__language-container > #trigger-dropdown-language-chooser {
```

Comment:
Ca we add a comment on why we need this CSS? I supose is for the language menu on top but it there's a comment and in the future Decidim solves this, we will remove this.